libraries/RC_Channel/RC_Channel.h

```diff
@@ -208,11 +208,13 @@ class RC_Channel {
         // inputs from 200 will eventually used to replace RCMAP
         ROLL =               201, // roll input
         PITCH =              202, // pitch input
-        WALKING_HEIGHT =     203, // walking robot height input
+        THROTTLE =           203, // throttle pilot input
+        YAW =                204, // yaw pilot input
```

Comment:
might as well also add forward and lateral while were at it? I think those 6 where why mainsail is at 207

Reply:
Good catch.... except that it looks like they're actually 6 and 7 :-(